ui/lesson/Terminal.tsx

```diff
@@ -21,27 +23,23 @@ export default function Terminal({ success, lines, next, onChange }) {
     >
       <div className="flex grow flex-col items-stretch text-white">
         <div className="flex grow">
-          <ReactTerminal
-            name="Saving Satoshi"
-            colorMode={ColorMode.Light}
-            onInput={onChange}
-          >
+          <TerminalInterface onInput={onChange}>
```

Comment:
I think there are a little too much Terminal-related components. I feel the logic that now resides in `TerminalInterface` could just be added to this component (`ui/lesson/Terminal.tsx`).